src/godot_fmod.cpp

```diff
@@ -1378,7 +1382,8 @@ void Fmod::unmute_all_events() {
 
 bool Fmod::banks_still_loading() {
     for (int i = 0; i < banks.size(); i++) {
-        FMOD::Studio::Bank* bank = banks.get(i);
+        auto key = banks.keys()[i];

```

Comment:
Godot codebase convention is to avoid using `auto` keyword in C++. Not sure if the `fmod-gdnative` team cares about matching Godot core's style conventions, but figured I'd mention it.

Reply:
I agree on this, if it is not a cast, we should avoid ˋauto`